test/web.js

```diff
@@ -10,710 +10,801 @@ var testCreds = require("./testCredentials");
 var url = require("url");
 
 lab.experiment("OAuth", function() {
-  var s = server({
-    loginAPI: "http://localhost:3232",
-    cookieSecret: "test",
-    oauth_clients: testCreds.clients,
-    authCodes: testCreds.authCodes,
-    accessTokens: testCreds.accessTokens,
-    enableCSRF: false
-  });
+  var s, s2, ls;
+  const cookieSecret = "test".padEnd(32, "test");
```

Comment:
I changed the cookie secret here since the new hapi cookie secret requirements enforce a minimum of 32 characters.